### PR TITLE
Add liveness check to agent charts

### PR DESCRIPTION
**Pull request by epot**
So k8s can automatically replace pods.

### CODE REVIEW

File: charts/warpstream-agent/templates/deployment.yaml [hunk 1]
```diff
@@ -63,6 +63,12 @@ spec:
               port: http
             initialDelaySeconds: 10
             periodSeconds: 5
+          livenessProbe:
+            httpGet:
+              path: /v1/status
+              port: http
+            initialDelaySeconds: 10
+            periodSeconds: 5
```
Comment (richardartoul) [inline charts/warpstream-agent/templates/deployment.yaml:71]: Whats the default number of allowed failures here?

**Review by richardartoul — APPROVED** [review]

File: charts/warpstream-agent/templates/deployment.yaml (hunk 1 above)
Comment (epot) [reply charts/warpstream-agent/templates/deployment.yaml:71]: 3. You think it's too aggressive? I tried deploying an agent on minikube and it worked fine but it was not talking to a blob storage.

Comment (richardartoul) [reply charts/warpstream-agent/templates/deployment.yaml:71]: 5s timeout? Thats probably fine tbh. Could do 5 if we're really paranoid with 5s timeout

Comment (epot) [reply charts/warpstream-agent/templates/deployment.yaml:71]: putting 5, it's cheap

**Pull request merged**